lib/Crust/Test.pm6

```diff
@@ -0,0 +1,81 @@
+use v6;
+unit class Crust::Test;
+
+our $Impl;
+$Impl ||= %*ENV<CRUST_TEST_IMPL> || "MockHTTP";
+
+method create(Crust::Test:U: Callable $app, *@args) {
+    my $subclass = "Crust::Test::$Impl";
+    require ::($subclass);
+
+    # do we need this?
+    if my $test-psgi = &::($subclass ~ "::test-psgi") {
```

Comment:
i will remove this.

On Friday, October 9, 2015, Tokuhiro Matsuno <notifications@github.com>
wrote:

> In lib/Crust/Test.pm6
> <https://github.com/tokuhirom/p6-Crust/pull/13#discussion_r41576947>:
>
> > @@ -0,0 +1,81 @@
> > +use v6;
> > +unit class Crust::Test;
> > +
> > +our $Impl;
> > +$Impl ||= %*ENV<CRUST_TEST_IMPL> || "MockHTTP";
> > +
> > +method create(Crust::Test:U: Callable $app, *@args) {
> > +    my $subclass = "Crust::Test::$Impl";
> > +    require ::($subclass);
> > +
> > +    # do we need this?
> > +    if my $test-psgi = &::($subclass ~ "::test-psgi") {
>
> what's this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tokuhirom/p6-Crust/pull/13/files#r41576947>.
>